### ISSUE TITLE
Bio2 Crash Logs

### ISSUE THREAD

**Issue by whogework** [gir489returns/Bioshock2CrashFix#3]
[crash.log](https://github.com/user-attachments/files/17370294/crash.log)

Crash was instant instead of the usual stuttering before the crash.

**Comment by gir489returns** [comment]
The log says you're crashing within my module (Bioshock2CrashFix.dll) at offset 0x1056.

Which is this: 
`.text:10001056                 cmp     dword ptr [eax], 0`

The fault code you're getting is 0xC0000005 which means the pointer is invalid.

However, according to SpecialK, the register contains 0x04B70000. That to me seems like a perfectly valid address?

Are you able to reproduce the issue? Or was this a one time "go fix it douchebag" thing.

**Comment by whogework** [comment]
I wasn't submitting this as a "fix it not my problem". I had crashed 3 times when trying to hit the spacebar after loading a save. This log was for a crash during a fight with the first Brute Splicer in Pauper's Drop. I have since cleared the fix files, verified integrity, and re-installed the fix. The integrity verification came back with some corrupt or missing files so it was most likely something on my end as I have been playing for about 30ish minutes with no issues. Sorry to waste your time I definitely should've looked further into it on my end first.
